features/search.feature

```diff
@@ -11,41 +11,3 @@ Feature: Searching
     Given I am on the home page
     When I search for something relevant
     Then I should see search results
-
-  Scenario: A relevant message is displayed when no results are found
-    Given I am on the home page
-    When I search for something irrelevant
-    Then I should see no search results
-    And I should prompted to try another search term
-
-  Scenario: Display spelling suggestion when search terms are misspelled
-    Given I am on the home page
-    When I search for an incorrectly spelt term with results
-    Then I should have search results
-    And I should see a spelling suggestion
-
-  Scenario: Correct search when search terms are misspelled
-    Given I am on the home page
-    When I search for an incorrectly spelt term with no results
-    Then I should have corrected search results
-
-  Scenario: A relevant message is displayed if a search is performed with no keywords
-    Given I am on the home page
-    When I submit a search with no query
-    Then I should see the search page
-
-  Scenario: Browsing paginated results
```

Comment:
should this scenario not be kept?

Reply:
This browsing pagination features were totally Google based so they kind need of a rewrite to be more flexible. I will open a separate PR today containing the new feature tests for browing and pagination.